### DISCUSSION TITLE
Astro components vs MDX

### DISCUSSION THREAD

**Discussion by jylin** [withastro/docs#2171] [Ideas]
Question: are there things you can do with astro components that you couldn't do with MDX? Is it worth considering deprecating the astro file format and just switching to MDX instead? There's far more support for MDX than Astro (for example, there's no IntelliJ support for *.astro files).

Also: is it possible to use MDX files for layouts? I think it currently doesn't support <slot/> in MDX files.

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
